tomviz/ImageStackDialog.cxx

```diff
@@ -0,0 +1,37 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+
+#include "ImageStackDialog.h"
+#include "ui_ImageStackDialog.h"
+
+#include "ImageStackModel.h"
+
+#include <QDialog>
```

Comment:
You have several includes that are duplicated that I only just noticed. General preferred layout - group includes together, class you derive from at the top, blank link, then Tomviz includes, then VTK, then ParaView, then Qt, then STL, not super strict, but grouping them helps. We alphabetize and clang enforces that.

Here the header already included QDialog, so no need to do so again here. The ImageStackModel is duplicated, in this case I would prefer declaring the class in the header, and including its header in this implementation file.